ios/Classes/WalletHandler.swift

```diff
@@ -18,9 +18,9 @@ class WalletHandler {
         "NEAR"  : NEAR(),
         "XLM"   : XLM(),
         "BNB"   : BNB(),
-        "BSC"   : BSC(),
+        "BSC"   : ETH(),
         "DOGE"  : DOGE(),
-        "MATIC" : MATIC(),
+        "MATIC" : ETH()
```

Comment:
Let's remove this then and just always use ETH . So all ETH compatible will just use the ETH